measureimagefocus.py

```diff
@@ -12,7 +12,7 @@
 import matplotlib.patches
 
 __doc__ = """
-For installation instructions and platform support notes, please read measureimagefocus_README.md.
+For installation instructions and platform support notes, please see https://github.com/CellProfiler/CellProfiler-plugins/wiki/Measure-Image-Focus.
```

Comment:
You can create a hyperlink:

```
External hyperlinks, like `Python <http://www.python.org/>`_. 
```

https://github.com/ralsina/rst-cheatsheet/blob/master/rst-cheatsheet.rst